dist/src/main/java/io/camunda/zeebe/shared/management/FlowControlEndpoint.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.shared.management;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.endpoint.web.WebEndpointResponse;
+import org.springframework.boot.actuate.endpoint.web.annotation.RestControllerEndpoint;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+@Component
+@RestControllerEndpoint(id = "flowControl")
+public class FlowControlEndpoint {
+
+  final FlowControlService flowControlService;
+
+  @Autowired
+  public FlowControlEndpoint(final FlowControlService flowControlService) {
+    this.flowControlService = flowControlService;
+  }
+
+  @PostMapping()
+  public ResponseEntity<?> post(@RequestBody final String flowControlCfg) {
```

Comment:
:wrench: Could this take a `FlowControlCfg` directly, instead of a string that is then later parsed?
Same for the get endpoint and the `FlowControlService`. Ideally these would all deal with `FlowControlCfg` instead of `String`

Reply:
Here in the get configuration we still retrieve a string because we don't save the configuration values in the FlowControl class just the limits with the already calculated values. Is it worth storing FlowControlCfg additionally here so we can fetch these values more cleanly?

Reply:
No I think it's okay that we return `Map<LimitType, Limit>` and not `FlowControlCfg` but I think it would be better to also return the actual types and not serialize to string too early. This would also solve the actuator decoder issue I think.